.luacheckrc

```diff
@@ -0,0 +1,5 @@
+redefined = false
+globals = {'box', 'utf8', 'checkers', '_TARANTOOL'}
+include_files = {'**/*.lua', '*.luacheckrc', '*.rockspec'}
+exclude_files = {'**/*.rocks/'}
+max_line_length = 120
```

Comment:
Could you reformat it with one table item per line?

```lua
redefined = false
globals = {
    'box',
    'utf8',
    'checkers',
    '_TARANTOOL'
}
include_files = {
    '**/*.lua',
    '*.luacheckrc',
    '*.rockspec'
}
exclude_files = {
    '**/*.rocks/'
}
max_line_length = 120
```

Added a patch on top of your branch.

Reply:
Also our Lua Style guide allows to ignore these warnings:
```
"212/self",   -- Unused argument <self>.
"411",        -- Redefining a local variable.
"421",        -- Shadowing a local variable.
"431",        -- Shadowing an upvalue.
"432",        -- Shadowing an upvalue argument.
```
And you may suppress them in config from the beginning.
See https://www.tarantool.io/en/doc/latest/dev_guide/lua_style_guide/#luacheck

Reply:
fixed